plugin-parser/mlops-plugin-parser.cc

```diff
@@ -88,10 +88,128 @@ typedef enum {
 } mlsvc_json_type_e;
 
 /**
- * @brief Parse json and update ml-service database via invoking daemon.
+ * @brief Internal enumeration for package manager event types.
+ */
+typedef enum {
+  MLSVC_PKGMGR_MDPARSER_PLUGIN_EVENT_TYPE_INSTALL = 0,
+  MLSVC_PKGMGR_MDPARSER_PLUGIN_EVENT_TYPE_UNINSTALL = 1,
+  MLSVC_PKGMGR_MDPARSER_PLUGIN_EVENT_TYPE_UPGRADE = 2,
+  MLSVC_PKGMGR_MDPARSER_PLUGIN_EVENT_TYPE_MAX
+} mlsvc_package_manager_event_type_e;
+
+/**
+ * @brief Internal function for uninstall models and pipelines.
  */
 static void
-_parse_json (JsonNode *node, mlsvc_json_type_e json_type, const gchar *app_info)
+_uninstall_rpk (const char *name, char *_info, mlsvc_json_type_e json_type)
```

Comment:
param _info : pass const char *_info

Reply:
thanks, i update it. ``char *_info`` -> ``const char *_info``